product_docs/docs/pgd/5/cli/command_ref/index.mdx

```diff
@@ -13,35 +13,65 @@ manage your EDB Postgres Distributed cluster. It allows you to run commands
-|-------|---------------|-----------------------------------------------------------------------------------------------|
-|       | --dsn         | Database connection string<br/>For example "host=bdr-a1 port=5432 dbname=bdrdb user=postgres"             |
-| -f    | --config-file | Name/Path to config file.<br/>This is ignored if --dsn flag is present<br/>Default "/etc/edb/pgd-cli/pgd-cli-config.yml" |
-| -h    | --help        | Help for pgd - will show specific help for any command used                                                                                 |
-| -L    | --log-level   | Logging level: debug, info, warn, error (default "error")                                     |
-| -o    | --output      | Output format: json, yaml                                                                     |
-
-## See also
-
-- [check-health](pgd_check-health)
-- [create-proxy](pgd_create-proxy)
-- [delete-proxy](pgd_delete-proxy)
-- [set-group-options](pgd_set-group-options)
-- [set-node-options](pgd_set-node-options)
-- [set-proxy-options](pgd_set-proxy-options)
-- [show-clockskew](pgd_show-clockskew)
-- [show-events](pgd_show-events)
-- [show-groups](pgd_show-groups)
-- [show-nodes](pgd_show-nodes)
-- [show-proxies](pgd_show-proxies)
-- [show-raft](pgd_show-raft)
-- [show-replslots](pgd_show-replslots)
-- [show-subscriptions](pgd_show-subscriptions)
-- [show-version](pgd_show-version)
-- [switchover](pgd_switchover)
-- [verify-cluster](pgd_verify-cluster)
-- [verify-settings](pgd_verify-settings)
+| Short | Long&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Description                                                                                                              |
+|-------|------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| -f    | --config-file                                                                | Name/Path to config file.<br/>This is ignored if --dsn flag is present<br/>Default "/etc/edb/pgd-cli/pgd-cli-config.yml" |
+|       | --dsn                                                                        | Database connection string<br/>For example "host=bdr-a1 port=5432 dbname=bdrdb user=postgres"                            |
+| -h    | --help                                                                       | Help for pgd - will show specific help for any command used                                                              |
+| -o    | --output                                                                     | Output format: `json`, `ascii`, `psql`, `modern`, `markdown-table`, `default`  (see [Output formats](#output-formats))                                          |
```

Comment:
`markdown-table` was renamed to `markdown`. `default` not a valid option any more